src/main/java/io/opentraffic/reporter/Segment.java

```diff
@@ -16,33 +16,18 @@
   public static long INVALID_SEGMENT_ID = 0x3fffffffffffL;
   public long id;         //main segment id
   public long next_id;    //optional next, could be invalid
-  public long min, max;   //epoch seconds
```

Comment:
when porting some of this logic to python i noticed that there was still vestigial code in there that was both wasting space and made for reading of the code a bit confusing so i cleaned it up for my understanding and others